src/Framework/FieldsAPI/FormField/FieldEmailTag.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\FormField;
+
+trait FieldEmailTag {
+
+	/** @var string */
+	public $emailTag;
```

Comment:
Please make this protected.

Reply:
Resolved by https://github.com/impress-org/givewp/pull/5649/commits/c5a1dff9785c1583dd6d4e7df84a54d2bb602da8